dipy/nn/utils.py

```diff
@@ -4,6 +4,31 @@
 from dipy.align.reslice import reslice
 from dipy.testing.decorators import warning_for_keywords
 
+from tensorflow import pad as tf_pad
```

Comment:
I think the errors are coming from here. Can you change it to only import if tensorflow installed like you have it on the other files?